Cargo.toml

```diff
@@ -8,6 +8,6 @@ license = "BSD-3-Clause"
 description = "ASN.1 (DER) serializer for Rust."
 
 [dependencies]
-byteorder = "*"
+byteorder = ">= 0.5.0"
```

Comment:
This way, an incompatible release 0.6.0 or 1.0.0 would break rust-asn1. Version pinning (to 0.5.x) is less of a problem than a project not building anymore.